Packs/Okta/Playbooks/playbook-Okta_-_User_Investigation.yml

```diff
@@ -1302,6 +1310,49 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: c0251946-a36e-4c1f-842c-95fe5aecc6cd
+    type: condition
+    task:
+      id: c0251946-a36e-4c1f-842c-95fe5aecc6cd
+      version: -1
+      name: The login country is defined?
```

Comment:
Remove the "The"
You can also rename to "Check if the login country is defined" (like the description)

Reply:
Done.